train.py

```diff
@@ -1,53 +1,16 @@
-from comet_ml import Experiment
-
-from argparse import ArgumentParser
 from pathlib import Path
 from time import time
 
+import hydra
+import yaml
 from addict import Dict
+from comet_ml import Experiment
```

Comment:
I remembered that the `from comet_ml import Experiment` needs to be the first line. Correct me if I am wrong.

Reply:
It does not need to be first but it does need to be before `torch` and other tensor stuff that's true